src/test/rgw/rgw_multi/tests.py

```diff
@@ -97,6 +89,17 @@ def meta_sync_status(zone):
 
     return period, realm_epoch, num_shards, markers
 
+def meta_sync_status(zone):
+    for _ in range(60):
```

Comment:
yeah, i just settled on a total of 5 minutes. i'll make this configurable along with the sleep in `set_master_zone()`